README.md

```diff
@@ -145,16 +145,15 @@ Build a Go server that receives log messages in the format defined in `proto/log
     1. Add a binary target from `server.go`
         - use [`go_binary()`][go_binary]
         - name it `go-server`
-        - it depends on `server_lib`
-        - add `//proto/logger:logger_go_proto` to `deps`
+        - it should be compiled together with the sources of `server_lib`
```

Comment:
I'm not sure I understand this change. The intention was to add an `embed` attribute to capture the `:server_lib` target.

Reply:
I found, the instructions make it sound like `:server_lib` goes into `deps`. So I looked for a way to make it more clear. I rewrote it this way after reading the following in the [`go_binary` documentation](https://github.com/bazelbuild/rules_go/blob/master/docs/go/core/rules.md#go_binary).
> **embed**: List of Go libraries whose sources should be compiled together with this binary's sources. Labels listed here must name `go_library`, `go_proto_library`, or other compatible targets with the [GoLibrary](https://github.com/bazelbuild/rules_go/blob/master/go/providers.rst#GoLibrary) and [GoSource](https://github.com/bazelbuild/rules_go/blob/master/go/providers.rst#GoSource) providers. Embedded libraries must all have the same `importpath`, which must match the `importpath` for this `go_binary` if one is specified. At most one embedded library may have `cgo = True`, and the embedding binary may not also have `cgo = True`. See [Embedding](https://github.com/bazelbuild/rules_go/blob/master/docs/go/core/embedding.md#embedding) for more information.

Reply:
Oh, I see. Hmm, yes, in that context that explanation makes sense. Thanks!